src/java/org/apache/cassandra/net/Verb.java

```diff
@@ -159,22 +159,23 @@ public enum Verb
     SCHEMA_VERSION_REQ     (20,  P1, rpcTimeout,      MIGRATION,         () -> NoPayload.serializer,                 () -> SchemaVersionVerbHandler.instance,   SCHEMA_VERSION_RSP  ),
 
     // repair; mostly doesn't use callbacks and sends responses as their own request messages, with matching sessions by uuid; should eventually harmonize and make idiomatic
+    // for the repair messages that implement retry logic, use rpcTimeout so the single request fails faster, then retries can be used to recover
     REPAIR_RSP             (100, P1, repairTimeout,   REQUEST_RESPONSE,  () -> NoPayload.serializer,                 () -> ResponseVerbHandler.instance                             ),
-    VALIDATION_RSP         (102, P1, longTimeout,     ANTI_ENTROPY,      () -> ValidationResponse.serializer,        () -> RepairMessageVerbHandler.instance,   REPAIR_RSP          ),
```

Comment:
This previously had a long timeout...
```
static final ToLongFunction<TimeUnit> longTimeout     = units -> Math.max(DatabaseDescriptor.getRpcTimeout(units), units.convert(5L, TimeUnit.MINUTES));
```
...because the validation response could have a large payload/Merkle trees, and we didn't want to time out too aggressively? I'm not sure what the impact of that will be, although we have talked about the fact that we might want to retry less aggressively for this verb...?

Reply:
we fall back to `longTimeout` now when disabled